src/main.zig

```diff
@@ -284,7 +286,7 @@ fn drawCenteredText(t: *Tetris, text: []const u8) void {
     drawText(t, text, draw_left, draw_top, 1.0);
 }
 
-fn draw(t: *Tetris) void {
+fn draw(t: *Tetris) !void {
```

Comment:
I don't think draw should have the possibility of failure.

Reply:
As you say

Reply:
As you say